Dockerfile

```diff
@@ -158,14 +162,12 @@ RUN --mount=type=cache,target=/root/.cache/microdnf:rw \
 # need to upgrade pip and install wheel before installing grpcio, before installing tensorflow on aarch64
 # use caching to speed up multi-platform builds
 ENV PIP_CACHE_DIR=/root/.cache/pip
-RUN --mount=type=cache,target=/root/.cache/pip \
```

Comment:
Same here. Do we need to delete cache part here? even ENV set `PIP_CACHE_DIR` ?

Reply:
we might need to delete them, or just keep it.
but I am not sure what the benefits are for this cache even for dev builds.

wdyt?

Reply:
The cache is for making builds faster when repeatedly building the container. It won't download again the dependencies once you run it once.

If the cache is not breaking Konflux, I don't think it should be removed.

Reply:
re-added